### ISSUE TITLE
Create README

### ISSUE THREAD

**Issue by jopopcorn** [euphony-io/soundless-check-in#2]
### Create README
* Draw up README explaining how to introduce and use the demo app.

**Comment by aiclaudev** [comment]
I think the configuration below would be appropriate.

This is a check-in application that identifies visitors, especially effective during the COVID-19.
It is based on Euphony library. 
Official Facebook Page : https://www.facebook.com/euphonyproject
Official Library Site : https://dev.jbear.co/euphony

###**Reason for making this application**
***
The existing 'QR code check-in'  has the following problems.

*If the lens of the device receiving the data is dirty, it is not recognized well.
*If the surroundings are too bright or too dark, it is not recognized well.
*The process of accessing QR codes is complicated, Especially for the elderly.

But, This application has no such inconvenience as using sound wave communication.

###**Run the app**
***

Images or video ...

###**Report Issues**
***
To report a specific problem or feature request, [please open a new issue](https://github.com/euphony-io/soundless-check-in/issues/new).


Please comment if there is anything that needs to be added or modified!

**Comment by JuSeongPil** [comment]
I make some change the config for the customer's view
​
# Soundless-check-in
​
![euphony_logo](https://user-images.githubusercontent.com/8490058/129358383-14b6f34d-9264-4c49-9538-3ef2e62a6490.png)
​
Euphony Speaker demo app generates noiseless sound wave to send the data.
When transferring data, the 'Speaker' generates an inaudible frequency and the 'Listener' receives it.

This is a check-in application that replace the QR code.
The QR code is hard for adults to using their mobile.
And this soundless check-in application can identifies the visitors following 3 information
-1. There Phone number
-2. There Residential Area
-3.The store that they visit


Follow the steps below and experience sound communication !
​
## Reason for making this application
​
The existing 'QR code check-in'  has the following problems.
​
* If the lens of the device receiving the data is dirty, it is not recognized well.
* If the surroundings are too bright or too dark, it is not recognized well.

* The process of accessing QR codes is complicated, especially for the elderly.


But, This application has no such inconvenience as using sound wave communication.


## Run the app



Images or video ...


## Report Issues


To report a specific problem or feature request, [please open a new issue](https://github.com/euphony-io/soundless-check-in/issues/new).


## Library


It is based on Euphony library.

Official Facebook Page : https://www.facebook.com/euphonyproject

Official Library Site : https://dev.jbear.co/euphony







Please comment free:)

**Comment by sujin9** [comment]
I have a few ideas!

First,
> Euphony Speaker demo app generates noiseless sound wave to send the data.
> When transferring data, the 'Speaker' generates an inaudible frequency and the 'Listener' receives it.  
  
I think it's better to delete these two sentences  because it's related to speaker demo app than our service.
Instead, we can indicate that we use euphony library for this app.

Second, 
How about add 'What's next' category?
We can write something can be updated in this section later !
지금은 샘플앱이지만 euphony 라이브러리가 업데이트 됨에 따라 추가하면 좋겠다 하는 기능
혹은 앞으로 이렇게 발전되면 좋겠다 하는 부분을 추가하면 어떨까요?

Please let me know your opinion on this :)

**Comment by jopopcorn** [comment]
> First,
> 
> > Euphony Speaker demo app generates noiseless sound wave to send the data.
> > When transferring data, the 'Speaker' generates an inaudible frequency and the 'Listener' receives it.
> 
> I think it's better to delete these two sentences because it's related to speaker demo app than our service.
> Instead, we can indicate that we use euphony library for this app.

I agree with both of Sujin's opinions! I think deleting this sentence would go well with the context.

> Second,
> How about add 'What's next' category?
> We can write something can be updated in this section later !
> 지금은 샘플앱이지만 euphony 라이브러리가 업데이트 됨에 따라 추가하면 좋겠다 하는 기능
> 혹은 앞으로 이렇게 발전되면 좋겠다 하는 부분을 추가하면 어떨까요?

And I think it's a good idea to write down the features and improvements we need to consider development expansion.

**Comment by designe** [comment]
Could you check one more review in `README.md`

for example,
```
But, This application has no such inconvenience as using sound wave communication.
```
```
If you press the 'GET INFO !', you can receive visitors' information.
```

**Comment by aiclaudev** [comment]
> Could you check one more review in `README.md`
> 
> for example,
> 
> ```
> But, This application has no such inconvenience as using sound wave communication.
> ```
> 
> ```
> If you press the 'GET INFO !', you can receive visitors' information.
> ```


I changed sentence you mentioned.
```
Because this application is user-friendly and use sound-communication, there is no inconvenience like above !
```

```
If you press the 'GET INFO!', this application will be state that can recognize visitor's information.
```

Is it okay? Really appreciate for your feedback !!

**Comment by designe** [comment]
@aiclaudev 
Thanks for the quick fix and reflection.
I once again asked for a review from the team members, because a review can make a better project! 
You have done well so far and I think you will do even better in the future :)

**Comment by jopopcorn** [comment]
@designe Thank you for ur detailed feedback!
Lastly, I'll review it and leave the comment if there is anything to modify.

**Comment by jopopcorn** [comment]
> # Soundless-check-in
> ![euphony_logo](https://user-images.githubusercontent.com/8490058/129358383-14b6f34d-9264-4c49-9538-3ef2e62a6490.png)
> 
> This is a check-in application that identifies visitors, especially effective during the COVID-19.
> This application stores below informations in database
> 
> > Visitor's phone number
> > Visitor's residential area
> > Information of company that they visit
> 

It was difficult to review quickly yesterday, so I'm commenting late.
I think this it would be good to change this phrase as follows.
I think the rest was well reflected based on the mentor's feedback!
Thank you for your effort!


</br>

# Soundless-check-in
This is a check-in application that identifies visitors, especially effective during the COVID-19.
You can check in with sound wave communication instead of QR code.
When the check-in is complete, visitor and store information are stored in the database, along with the check-in time.

**Comment by aiclaudev** [comment]
> > # Soundless-check-in
> > ![euphony_logo](https://user-images.githubusercontent.com/8490058/129358383-14b6f34d-9264-4c49-9538-3ef2e62a6490.png)
> > This is a check-in application that identifies visitors, especially effective during the COVID-19.
> > This application stores below informations in database
> > > Visitor's phone number
> > > Visitor's residential area
> > > Information of company that they visit
> 
> @aiclaudev
> 
> It was difficult to review quickly yesterday, so I'm commenting late.
> I think this it would be good to change this phrase as follows.
> I think the rest was well reflected based on the mentor's feedback!
> Thank you for your effort!
> 
> 
> # Soundless-check-in
> This is a check-in application that identifies visitors, especially effective during the COVID-19.
> You can check in with sound wave communication instead of QR code.
> When the check-in is complete, visitor and store information are stored in the database, along with the check-in time.

Thanks ! If the existing sentence is changed by your suggestion, README.md readers will be able to understand the app more easily. Thank you.
